lib/smoochApiBot.js

```diff
@@ -22,6 +22,23 @@ class SmoochApiBot extends Bot {
         });
         return api.appUsers.sendMessage(this.userId, message);
     }
+
+    setTypingActivity(typing) {
```

Comment:
Instead of the if, could you set a parameter default? Eg: `setTypingActivity(typing = true) {`